tests/jerry/regression-test-issue-3814.js

```diff
@@ -0,0 +1,17 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
The test case should be in the "es2015" sub-folder.

Reply:
Indeed! Thanks for the remark, I've updated the PR.